creduce/get_key.py

```diff
@@ -0,0 +1,40 @@
+import time
+import sys
+import sys
+import tty
+import termios
+import select
+
+def get_key():
+
+    (ready, _, _) = select.select([fd], [], [], 0)
+
+    if ready:
+        key = sys.stdin.read(1)
+    else:
+        key = None
+
+    return key
+
+if __name__ == "__main__":
+    fd = sys.stdin.fileno()
+    old_settings = termios.tcgetattr(fd)
+    tty.setraw(sys.stdin.fileno())
+
+    i = 0
+
+    while True:
```

Comment:
It's not an absolute must-have, but on the other hand I'd been planning on adding some more keyboard shortcuts such as those appearing in @vgvassilev's pull req that toggles the print-diffs mode.